src/SquidConfig.h

```diff
@@ -505,7 +505,7 @@ class SquidConfig
     struct {
         Security::FuturePeerContext *defaultPeerContext;
         // TODO: Remove when FuturePeerContext above becomes PeerContext
-        Security::ContextPointer sslContext;
+        Security::ContextPointer *sslContext;
```

Comment:
I noticed that this field only shows up in construction/destruction code diff. Let's confirm and ban its use elsewhere:

```suggestion
        /// \deprecated Legacy storage. Use defaultPeerContext instead.
        Security::ContextPointer *sslContext_;
```

... and then adjust the rest of PR code to compile, of course.

Reply:
Done (dc66a68).